app/controllers/application_controller.rb

```diff
@@ -1,6 +1,22 @@
 class ApplicationController < ActionController::Base
   helper_method :current_user_id, :allow_user_managers_service_access?
 
+  class ForbiddenError < StandardError; end
+
+  rescue_from ForbiddenError do |_e|
+    render status: :forbidden, template: 'errors/forbidden'
+  end
+
+  def landing_page_for(user)
+    if user.user_manager?
+      admin_manage_users_root_path
+    elsif user.service_user?
+      assigned_applications_path
+    elsif user.reporting_user?
+      reports_path
```

Comment:
Supervisors are both service and reporting users. This would route them to the assigned applications page, just confirming that is correct or do we want them to land on the reports page?